crypto/bio/bio_meth.c

```diff
@@ -10,7 +10,7 @@
 #include "bio_local.h"
 #include "internal/thread_once.h"
 
-CRYPTO_REF_COUNT bio_type_count;
+CRYPTO_REF_COUNT bio_type_count = { .val = 0 };
```

Comment:
Our requirement is C89 *syntax* for common code.  We have made exceptions for code that's properly fenced with preprocessor guards, but that will require a C89 compatible alternative anyway, and for this sort of thing, that's a waste of electrons.